scripts/build.mjs

```diff
@@ -23,8 +23,9 @@ const appimageX64 = 'appimage-x64';
 const dir = 'dir';
 const mac = 'mac';
 const snap = 'snap';
+const deb = 'deb';
 const windows = 'windows';
-const availableTargets = [appimage, appimageX64, dir, mac, snap, windows];
+const availableTargets = [appimage, appimageX64, dir, mac, snap, deb, windows];
```

Comment:
This list is ordered alphabetically, so `deb` should be next to `appimageX64`

Reply:
Done!